test/test_metrics.py

```diff
@@ -162,6 +163,22 @@ def test_metrics_report(self):
     report = met.metrics_report()
     self.assertIn("CachedCompile", report)
 
+  def test_execute_time_metric(self):
+    xm.xla_device()
+
+    begin = time.perf_counter_ns()
+    value = torch.randn(
+        10000, 10000, device=xm.xla_device()) * torch.randn(
+            10000, 10000, device=xm.xla_device())
+    value_mean = value.mean()
+    xm.mark_step()
+    cpu_value = value_mean.cpu()
+    wall_time_ns = time.perf_counter_ns() - begin
+    self.assertIn("ExecuteTime", met.metric_names())
+    execute_time_ns = met.metric_data('ExecuteTime')[1]
+    self.assertGreater(execute_time_ns, .5 * wall_time_ns)
+    print(execute_time_ns / 1e9, wall_time_ns / 1e9)
```

Comment:
Remove this print statement, since the values will already get printed if this fails.